src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/jpa/entity/DocumentsEntity.kt

```diff
@@ -22,7 +22,7 @@ class DocumentsEntity(
   val prisonerId: Long,
 
   @Column(name = "original_document_key")
-  val originalDocumentKey: UUID?,
+  var originalDocumentKey: UUID?,
```

Comment:
why has this been made mutable, can't see any assignments to it and it doesn't make sense to change the original document key

Reply:
oops! not realise how it got changed, reverted it!